libanswers/custom-footer.html

```diff
@@ -33,7 +33,7 @@ <h4><a href="https://libraries.mit.edu/research-support">Research support</a></h
 				<a href="https://libraries.mit.edu/experts" class="link-sub">Research guides &amp; expert librarians</a>
 				<a href="https://libraries.mit.edu/productivity-tools" class="link-sub">Productivity tools</a>
 				<a href="https://libraries.mit.edu/scholarly" class="link-sub">Scholarly publishing</a>
-				<a href="https://libraries.mit.edu/references" class="link-sub">Citation &amp; writing tools</a>
+				<a href="https://libraries.mit.edu/references" class="link-sub">Citation &amp; writing software</a>
```

Comment:
Not sure why this is "software" on the main site, but it really should be "tools" not "software"

Reply:
I just pushed a commit to make this change.